llvm/utils/lit/lit.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
You mean that this is annoying because invoking lit directly still uses Python 2, meaning that developers would only learn about Python 3 failures during PR testing, but not locally during development, right?

Reply:
(Discussed this elsewhere.)